pkg/setting/setting_unified_alerting.go

```diff
@@ -318,13 +319,15 @@ func (cfg *Cfg) ReadUnifiedAlertingSettings(iniFile *ini.File) error {
 	uaCfg.ReservedLabels = uaCfgReservedLabels
 
 	stateHistory := iniFile.Section("unified_alerting.state_history")
+	stateHistoryLabels := iniFile.Section("unified_alerting.state_history.external_labels")
```

Comment:
INI doesn't have arbitrarily deep nesting of objects like yaml/toml. So, I felt that an extra section was the most fluid way to represent this in the config file. We can see from the `defaults.ini` that there is precedent for doing this in a few other areas of Grafana.

Reply:
SGTM

Reply:
I'm not sure I love this - can you show an example of what the configuration looks like?

It's very common within Prometheus to have the following syntax for labels:

```
external_labels = lbl1=val1,lbl2=val3
```

So maybe that's an option?

Reply:
Sorry, I merged right as this comment landed and it was not visible to me. It looks like:
```
[unified_alerting.state_history.external_labels]
lbl1 = val1
lbl2 = val2
```
I considered the all-on-one-line approach above and did not do it because:
- It was less safe, the current approach is more clear and difficult to break, and labels can be merged at the INI merge level.
- The precedent that alerting has for that pattern _only lists label names_ not label key/value pairs. It's a different use case, and in that context, harder to break because there is less syntax/parsing involved.
- There are [other sections](https://github.com/grafana/grafana/blob/b4682fe3cbb37bf7efdaa3fe6645544782f36237/conf/defaults.ini#L1088) in the grafana config file that already use this format, so the current appraoch is also locally consistent.